src/Searcher.tsx

```diff
@@ -117,6 +118,12 @@ export class Searcher {
             return 0;
         });
 
+        for (let i = 0; i < leafItems.length; i++) {
+            leafItems[i].props.index = i;
+        }
+
+        this.libraryContainer.setMaxSelectionIndex(leafItems.length - 1);
```

Comment:
Do we need to reset selectionIndex as well? Otherwise, when search text changes, the default one will not be the 1st any more.